apps/user/README.md

```diff
@@ -47,3 +47,94 @@ http://localhost:8080/swagger
 ### unit tests
 
 `yarn test:unit`
+
+## How to access wallet api (Draft)
+
+### Set up Keycloak 
+
+Attach permission by adding new role to this user api client, so the wallet api can auth with this user api client by verifying the role, say: `wallet-operator-microservice`
+
+1. Create a new realm role
+  1.1. Go to `Realm Roles` -> `Create Role`
+  1.2. Name it `wallet-operator-microservice`
+  1.3. Save
+
+2. Attach the role to the user api client
+  2.1. Go to `Clients` -> `wallet-app-user-dev-svc` -> `Client scopes`
+  2.2. Find the item: `xxx-dedicated`, here in this case, it is `wallet-app-user-dev-svc-dedicated`
+  2.3. Click `Add mapper` -> `by configuration` -> `Hardcoded role`
+  2.4. Input `operator`, choose the role created in step 1, `wallet-operator-microservice`
+  2.5. Save
+
+
+### Get access token
+
+```
+curl -X POST "https://dev-k8s.treetracker.org/keycloak/realms/treetracker/protocol/openid-connect/token" \
+-H "Content-Type: application/x-www-form-urlencoded" \
+-d "grant_type=client_credentials" \
+-d "client_id=wallet-app-user-dev-svc" \
+-d "client_secret=xxx"
+```
+
+will get a response like:
```

Comment:
Should this be referred to as the decoded token response instead of the response for clarity?